src/common/snippets/src/lowered/pass/solve_buffer_memory.cpp

```diff
@@ -102,9 +102,8 @@ std::vector<ov::MemorySolver::Box> SolveBufferMemory::init_boxes(const Buffers&
     boxes.reserve(map_boxes.size());
     for (auto& p : map_boxes) {
         auto& box = p.second;
-        // We use data alignment to put data in the line cache
-        // TODO [143395] : Please check if alignment is really needed here
-        box.size = utils::div_up(box.size, m_alignment);
+        // Align with cache line size. The experiments shows that it affects performance.
+        box.size = utils::div_up(box.size, byte_alignment);
```

Comment:
When I validate performance of the current feature with 1 byte for `flag` in BrgemmCopyA, I saw the performance regressions when I allocate one extra byte. it's connected with cache line size (the majority of CPUs have 64 bytes but we don't have API to get real size). So I removed this comment (I'm planning to write the explanation in the ticket after review this part) , added alignment in RuntimeConfigurator: the unit tests are successfully passed (we allocate the same size as before).